lambda-layer/packages/layer/install-externals.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+set -euf -o pipefail
+
+# Space separated list of external NPM packages
+EXTERNAL_PACKAGES=( "import-in-the-middle" )
```

Comment:
can you add sufficient comment to this file, why we need it and what is it for

Reply:
done